vpc_peering/README.md

```diff
@@ -1,29 +1,115 @@
-# Terraform VPC Peering Module
+# Terraform VPC Peering Module Between Accounts
 
-Refer <https://registry.terraform.io/modules/grem11n/vpc-peering/aws/latest> which seems to be the best one
+## Resources
+
+* Tags
+
+## Input Variables
+
+| Name            | Type        | Default | Example                                                                         |
+|-----------------|-------------|---------|---------------------------------------------------------------------------------|
+| name_tag_value  | string      |         |                                                                                 |
+| user_tags       | map(string) |         | {   team    =  "devops"   purpose =  "vpc peering test"   owner   =  "naveen" } |
+| octopus_tags    | map(string) |         | {   project         =  "test"   space           =  "Default" }                  |
+
+For `user_tags` , refer <https://github.com/variant-inc/lazy-terraform/tree/master/submodules/tags>
+
+`octopus_tags` are auto set at octopus. Set the variable as
+
+```bash
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type = map(string)
+}
+```
 
 ## Examples
 
+versions.tf
+
+```bash
+terraform {
+  required_version = ">= 0.15.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.30"
+    }
+  }
+}
+```
+
+provider.tf
+
 ```bash
 provider "aws" {
+  alias   = "this"
```

Comment:
Can you make more clear what 'this' is. Either with comments or by changing the name to `this_account` or `this_vpc` and similarly with `peer` change to `peer_account` or `peer_vpc`

Reply:
Added more description. I have used same aliasing names just to keep naming consistent with the peering module we are using.